defaults/main.yml

```diff
@@ -613,7 +611,7 @@ verify_maas_retries: 5
 # maas_use_api: Allow operations that make use of the MaaS api, set to false
 #               for offline testing
 #
-maas_use_api: true
+maas_use_api: false
```

Comment:
Setting this to false here means that we'll have to override it to true in ``user_rpco_variables_defaults.yml``. Otherwise RPCO deployments won't register checks with the MaaS API by default.